package.json

```diff
@@ -15,25 +16,35 @@
     "test": "hardhat test"
   },
   "devDependencies": {
+    "@keep-network/hardhat-helpers": "^0.2.0-pre.4",
+    "@keep-network/hardhat-local-networks-config": "^0.1.0-pre.0",
     "@keep-network/prettier-config-keep": "github:keep-network/prettier-config-keep#d6ec02e",
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@tenderly/hardhat-tenderly": "^1.0.12",
+    "@types/chai": "^4.2.22",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.11.6",
     "chai": "^4.3.4",
     "eslint": "^7.27.0",
     "eslint-config-keep": "github:keep-network/eslint-config-keep#0c27ade",
-    "ethereum-waffle": "^3.4.0",
-    "ethers": "^5.4.1",
-    "hardhat": "^2.4.3",
+    "ethereum-waffle": "^3.0.0",
+    "ethers": "^5.0.0",
+    "hardhat": "^2.6.7",
+    "hardhat-deploy": "^0.9.4",
     "hardhat-gas-reporter": "^1.0.4",
+    "hardhat-local-networks-config-plugin": "^0.0.6",
     "prettier": "^2.3.2",
     "prettier-plugin-sh": "^0.7.1",
     "prettier-plugin-solidity": "^1.0.0-beta.14 ",
     "solhint": "^3.3.6",
     "solhint-config-keep": "github:keep-network/solhint-config-keep",
-    "typescript": "^4.3.2"
+    "ts-node": "^10.4.0",
+    "typescript": "^4.4.4"
   },
   "dependencies": {
-    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#507c647",
-    "@openzeppelin/contracts": "^4.3"
+    "@keep-network/keep-core": "^1.8.0-dev.4",
+    "@openzeppelin/contracts": "^4.3",
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#804cdbd"
```

Comment:
Let's bump it up to the `main` merge commit hash, that is https://github.com/thesis/solidity-contracts/commit/4985bcfc28e36eed9838993b16710e1b500f9e85. `804cdbd` points to a feature branch.

Reply:
Updated, see 613984b721ca714f414cfde7b191692d38c9c41f.